addons/point_of_sale/static/src/js/Screens/PartnerListScreen/PartnerListScreen.js

```diff
@@ -181,24 +179,15 @@ odoo.define('point_of_sale.PartnerListScreen', function(require) {
             if(this.state.query) {
                 domain = [["name", "ilike", this.state.query + "%"]];
             }
-            const result = await this.env.services.rpc(
-                {
-                    model: 'pos.session',
-                    method: 'get_pos_ui_res_partner_by_params',
-                    args: [
-                        [odoo.pos_session_id],
-                        {
-                            domain,
-                            limit: 10,
-                        },
-                    ],
-                    context: this.env.session.user_context,
-                },
-                {
-                    timeout: 3000,
-                    shadow: true,
-                }
-            );
+            const result = await this.env.services.orm
+                .withOptions({ timeout: 3000, silent: true })
```

Comment:
Hi @pimodoo. There are RPCs like this, where we pass `timeout` as option. Do you think we can remove it from everywhere? Can you give me a reason why we need to keep it?

Reply:
I don't know, maybe check the commit that introduced it to find why we put it there

Reply:
I need to keep the timeout feature, `pos_adyen` relies on it.